quasar_entities/src/ledger.rs

```diff
@@ -24,12 +26,32 @@ pub struct Model {
 }
 
 #[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]
-pub enum Relation {}
+pub enum Relation {
+    #[sea_orm(
+        has_many = "super::account::Entity",
+        to = "super::account::Column::LastModified",
+        from = "Column::Sequence"
+    )]
+    Account,
+}
+
+impl Related<super::account::Entity> for Entity {
+    fn to() -> RelationDef {
+        Relation::Account.def()
+    }
+}
 
 impl ActiveModelBehavior for ActiveModel {}
 
 #[ComplexObject]
-impl Model {}
+impl Model {
+    pub async fn accounts<'ctx>(&self, ctx: &Context<'ctx>) -> Result<Vec<account::Model>, DbErr> {
```

Comment:
We might need a pagination part two for these, they shouldnt load everything.

Reply:
Yes

Reply:
Will you be able to add it in your pagination PR after I merge this?